core/dbt/tests/fixtures/project.py

```diff
@@ -0,0 +1,270 @@
+import os
```

Comment:
This acts as a local plugin, and the idea here is that it could act as a plugin for the adapter tests too. Since the test fixtures and the dbt release are so tightly coupled, there would be overhead and coordination required to separate them out, and there aren't many advantages. Anybody using this file as a pytest plugin is also going to have to have dbt. Perhaps there will be considerations that make that necessary, but we're going to wait until we see how the adapter test rewrite plays out to make those decisions.